openapi.yaml

```diff
@@ -1,116 +0,0 @@
-#
```

Comment:
do we need to delete this? I get the desire to remove go-swagger and the code-gen, but we've had several requests for API documentation.

Reply:
I can bring it back, but we'd need to find a way to ensure it stays in sync with the code.  Any ideas?